core/load.py

```diff
@@ -31,13 +31,13 @@
 from core.compatible import copy_dir_tree
 from core.compatible import mkdir
 from core.compatible import get_module_dir_path
-from database.connector import insert_bulk_events_from_thread
-from database.connector import insert_events_in_bulk
+from database.connector import push_events_to_database_from_thread
+from database.connector import push_queues_to_database
```

Comment:
i think bulk_events tells you more than push_queues

Reply:
I couldn't understand what bulk_events meant until I read the entire code.

Reply:
okay then let's leave it! maybe @Ali-Razmjoo would have some suggestions later.

Reply:
I renamed. See if that helps.